src/main/asciidoc/serializers.adoc

```diff
@@ -0,0 +1,269 @@
+ifndef::sourcedir[]
+:sourcedir: ../..
+endif::sourcedir[]
+
+= Custom Serializers
+
+The Ehcache 3 documentation at http://www.ehcache.org/documentation/3.0/serializers-copiers.html#serializers[Serializers]
+gives you an overview of how to use custom serializers with a cache.
+The section on http://www.ehcache.org/documentation/3.0/serializers-copiers.html#persistent-vs-transient-caches[Persistent and Transient Serializers]
+briefly covers the serializer contracts that must be honored while writing custom serializers to be used with 
+persistent/transient caches.
+
+This document explains how you can write a transient/persistent custom serializer that works with Ehcache.
```

Comment:
How about "this article" instead of "this document".     Documents are more official part of product docs.   Blogs/articles are a bit less formal.